experiments/utils/download.py

```diff
@@ -134,7 +142,11 @@ def _create_zip(
     with zipfile.ZipFile(buffer, "a", zipfile.ZIP_DEFLATED, True) as zip_file:
         export_report = EXPORT_REPORT_HEADER
 
-        for dataPoint in experiment.datapoint_set.all():
+        qs = experiment.datapoint_set.all()
+        if queryset is not None:
+            qs = queryset
```

Comment:
This feels weird?

I'd rather set the ``queryset`` variable if it's ``None``:

```python
if queryset is None:
    queryset = experiment.datapoint_set.all()
```

Reply:
That was my first attempt, but the python type checker I use didn't like that I was iterating over an Optional variable. So I had to introduce a second variable whose type is not Optional, and that's why it should have a reasonable default.